run-pty.json

```diff
@@ -3,6 +3,8 @@
     "title": "Build much-select for development",
     "command": [
       "watchexec",
+      "--project-origin",
```

Comment:
tested locally and i can still run successfully with this so im good with these changes 👍🏻